TestResultSummaryService/parsers/Test.js

```diff
@@ -58,25 +58,36 @@ class Test extends Parser {
 
                 testStr = "";
                 testName = m[1];
+                console.log("testname!    " + testName)
                 testResultRegex = new RegExp(testName + "_(.*)\r?");
             }
             if ((m = line.match(regexStartTime)) !== null) {
-                startTime = m[1];
+                startTestName = m[1];
+                startTime = m[2];

```

Comment:
we need to validate the `startTestName` and `startTime ` > 0 before we can set `startTime `. We can use `testName ` variable in regex (similar to line 62) or check the value of `m[1]` contains `testName `.

Same for the `finishTime `